neoload/commands/report.py

```diff
@@ -0,0 +1,1056 @@
+import click
+from neoload_cli_lib import tools,cli_exception,user_data,displayer
+import neoload_cli_lib.schema_validation as schema_validation
+import jinja2
+import json
+import tempfile
+import os
+import webbrowser
+import sys
+import time
+import re
+import statistics
+
+import math
+import functools
+import datetime
+import uuid
+from threading import Lock
+
+from neoload_cli_lib import tools, rest_crud, user_data, displayer, cli_exception
+from neoload_cli_lib.name_resolver import Resolver
+
+from dateutil.relativedelta import relativedelta
+
+import logging
+import concurrent.futures
+import requests
+
+requests.adapters.DEFAULT_RETRIES = 5
+
+__endpoint = "/test-results"
+__operation_statistics = "/statistics"
+__operation_sla_global = "/slas/statistics"
+__operation_sla_test = "/slas/per-test"
+__operation_sla_interval = "/slas/per-interval"
+__operation_results_raw = "/raw"
+
+__operation_events = "/events"
+__operation_elements = "/elements"
+__operation_monitors = "/monitors"
+
+__resolver = Resolver(__endpoint, rest_crud.base_endpoint_with_workspace)
+
+meta_key = 'result id'
+gprint = print
+
+MAX_CALLS_PER_SECOND = max(1,math.floor(((200 - 50) / 60)))
+REQUEST_COUNT = 0
+
+@click.command()
+@click.option('--template', help="A built-in known report type or the file path to the .j2 template. Built-in types include:    'builtin:transactions-csv'     'builtin:transactions-json'")
+@click.option('--json-in', help="The file path to the .json data if previously stored using --out-file and no template or a json template")
+@click.option('--out-file', help="The file path to the resulting output, or none to print to stdout")
+@click.option('--filter', help="A filter statement to scope to a timespan and/or specify which reports and elements to use for multi-result analysis")
+@click.option('--max-rps', type=int, help="Explicit control over max concurrency of API calls")
+@click.option('--type', 'report_type',
+                type=click.Choice(['single','trends'], case_sensitive=False),
+                required=False, default="single")
```

Comment:
Add a description to say "default is single".

Reply:
Added help message in 0ac6fe6